app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/rootnav/RootNavViewModel.kt

```diff
@@ -122,6 +137,14 @@ class RootNavViewModel @Inject constructor(
                     SpecialCircumstance.VaultShortcut,
                     null,
                     -> RootNavState.VaultUnlocked(activeUserId = userState.activeAccount.userId)
+
+                    is SpecialCircumstance.CompleteRegistration ->
+                        RootNavState.CompleteOngoingRegistration(
+                            email = specialCircumstance.completeRegistrationData.email,
+                            verificationToken = specialCircumstance.completeRegistrationData.verificationToken,
+                            fromEmail = specialCircumstance.completeRegistrationData.fromEmail,
+                            timestamp = specialCircumstance.timestamp,
+                        )
```

Comment:
⚠️  This branch is unreachable. `CompleteRegistration` is being handled above, on line 70.